extensions/resource-deployment/package.nls.json

```diff
@@ -80,5 +80,12 @@
 	"sql-azure-database-server-display-name": "Database Server",
 	"azure-sqldb-agreement": "I accept {0}, {1} and {2}.",
 	"azure-sqldb-agreement-sqldb-eula": "Azure SQL DB License Terms",
-	"azure-sqldb-agreement-azdata-eula": "azdata License Terms"
+	"azure-sqldb-agreement-azdata-eula": "azdata License Terms",
+	"azure-sql-mi-display-name": "Azure SQL managed instance",
```

Comment:
Why would we have one version with managed instance capitalized and one without? That should generally always be consistent for product names.